### PR TITLE
DVCSMP-2168 Philips Hue/LIFX: Send device watch registration

### PR DESCRIPTION
**Pull request by larsfinander**
-Send device watch register events in update()

### CODE REVIEW

**Comment by larsfinander** [comment]
@jackchi @juano2310

**Comment by juano2310** [comment]
That won’t scale if we support more than one hub

File: smartapps/smartthings/hue-connect.src/hue-connect.groovy [hunk 1]
```diff
@@ -321,6 +322,14 @@ def uninstalled(){
 	state.username = null
 }
 
+private setupDeviceWatch() {
+	def hub = location.hubs[0]
```
Comment (juano2310) [inline smartapps/smartthings/hue-connect.src/hue-connect.groovy:326]: This won’t scale if we support more than one hub. We should track this.

File: smartapps/smartthings/lifx-connect.src/lifx-connect.groovy [hunk 2]
```diff
@@ -271,9 +271,17 @@ def initialize() {
 	updateDevices()
 	// Check for new devices and remove old ones every 3 hours
 	runEvery5Minutes('updateDevices')
+	setupDeviceWatch()
 }
 
 // Misc
+private setupDeviceWatch() {
+	def hub = location.hubs[0]
```
Comment (juano2310) [inline smartapps/smartthings/lifx-connect.src/lifx-connect.groovy:279]: This won’t scale if we support more than one hub. We should track this.

**Comment by juano2310** [comment]
Besides the potential issue with this being limited to only one hub I think the code looks good. 👍

File: smartapps/smartthings/lifx-connect.src/lifx-connect.groovy [hunk 3]
```diff
@@ -271,9 +271,17 @@ def initialize() {
 	updateDevices()
 	// Check for new devices and remove old ones every 3 hours
 	runEvery5Minutes('updateDevices')
```
Comment (jackchi) [inline smartapps/smartthings/lifx-connect.src/lifx-connect.groovy:273]: If `updated()` is called multiple times, will this schedule multiple `updateDevices`?

File: smartapps/smartthings/hue-connect.src/hue-connect.groovy (hunk 1 above)
Comment (jackchi) [reply smartapps/smartthings/hue-connect.src/hue-connect.groovy:326]: Will `it.device.hub` work here?

**Review by jackchi — COMMENTED** [review]
👍  DeviceWatch-Enroll event looks good. 
@juano2310

**Pull request merged**